### PR TITLE
DEP Pull

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Dimensions-Foundation/dimensions-foundation-web-theme/pull/27" target="_blank">ReviewNinja

File: style.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 * Author: Michael Kracke
 * Author URI: http://www.michaelkracke.com
 * Description: This is a theme built for Dimensions Education Programs
-* Version: 4.4.2.3.1
+* Version: 4.5.2.3.1
```
Comment (ghost) [inline style.css]: This should be 4.6 just to avoid future confusion !resolve

File: style.css [hunk 2]
```diff
@@ -6,14 +6,14 @@
 * Author: Michael Kracke
 * Author URI: http://www.michaelkracke.com
 * Description: This is a theme built for Dimensions Education Programs
-* Version: 4.4.2.3.1
+* Version: 4.6.2.3.1
```
Comment (ghost) [inline style.css:9]: Moved to 4.6.2.3.1 to avoid confusion with NEP branch !fixed

File: style.css (hunk 1 above)
Comment (ghost) [inline style.css]: !resolved

File: style.css (hunk 2 above)
Comment (ghost) [inline style.css:9]: !resolve

Comment (ghost) [inline style.css:9]: !resolved

**Pull request merged**